lib/machinery.rb

```diff
@@ -33,6 +33,7 @@
 require "find"
 require "pathname"
 require "nokogiri"
+require "etc"
```

Comment:
This is only used in the tests, isn't it? Then it should rather be included in `spec_helper.rb`.